rest/resource-server/src/docs/asciidoc/vulnerabilities.adoc

```diff
@@ -125,4 +125,12 @@ A `DELETE` request to delete a release vulnerability relation
 include::{snippets}/should_document_delete_release_vulnerability_relation/curl-request.adoc[]
 
 ===== Example response
-include::{snippets}/should_document_delete_release_vulnerability_relation/http-response.adoc[]
\ No newline at end of file
+include::{snippets}/should_document_delete_release_vulnerability_relation/http-response.adoc[]
+
+[[resources-vulnerabilitie-get-status]]
+==== Get a vulnerability tracking status
+
+A `GET` request will get vulnerability tracking status.
```

Comment:
Make the description more clear

Example: vulnerability tracking status for a project (or something)